tests/foreman/cli/test_errata.py

```diff
@@ -401,12 +910,66 @@ def test_positive_list_filter_by_product_id_and_org_id(self):
         1. erratum list --product-id=<product_id> --organization-id=<org_id>
 
         @Assert: Errata is filtered by product id and Org id.
-
-        @caseautomation: notautomated
-
         """
-
-    @stubbed()
+        product_erratum_list = Erratum.list({
+            'organization-id': self.org['id'],
+            'product-id': self.org_product['id'],
+            'per-page': 1000
+        })
+        product_errata_ids = {
+            errata['errata-id']
+            for errata in product_erratum_list
+        }
+        product_small_erratum_list = Erratum.list({
+            'organization-id': self.org_multi['id'],
+            'product-id': self.org_multi_product_small['id'],
+            'per-page': 1000
+        })
+        product_small_errata_ids = {
+            errata['errata-id']
+            for errata in product_small_erratum_list
+        }
+        product_big_erratum_list = Erratum.list({
+            'organization-id': self.org_multi['id'],
+            'product-id': self.org_multi_product_big['id'],
+            'per-page': 1000
+        })
+        product_big_errata_ids = {
+            errata['errata-id']
+            for errata in product_big_erratum_list
+        }
+        self.assertEqual(
+            len(product_errata_ids), self.org_product_erratum_count)
+        self.assertEqual(
+            len(product_small_errata_ids),
+            self.org_multi_product_small_erratum_count
+        )
+        self.assertEqual(
+            len(product_big_errata_ids),
+            self.org_multi_product_big_erratum_count
+        )
+        self.assertIn(self.org_product_errata_id, product_errata_ids)
+        self.assertIn(
+            self.org_multi_product_small_errata_id, product_small_errata_ids)
+        self.assertIn(
+            self.org_multi_product_big_errata_id, product_big_errata_ids)
+        self.assertEqual(
+            product_errata_ids.intersection(
+                product_small_errata_ids),
+            set([])
```

Comment:
I mentioned a lot of cases that is using this code. But if decide extract a method with common code, you could fix the code only on this extracted method.